src/redux/actions/postingActions.js

```diff
@@ -128,8 +129,11 @@ export const loadItemDetail = (itemId) => {
         ownerUsername,
       };
       dispatch(loadItemDetailSuccess(item));
+      return "success";
```

Comment:
it's fine for this commit, but should we consider constants for strings that represent response statuses in the future?